ci/docker/skaffold.yaml

```diff
@@ -0,0 +1,49 @@
+# (C) Copyright 2020-2023 Nuxeo (http://nuxeo.com/) and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: skaffold/v2beta18
+kind: Config
+build:
+  tagPolicy:
+    envTemplate:
+      template: "{{.VERSION}}"
+  artifacts:
+    - image: "$DOCKER_REGISTRY/nuxeo/nuxeo-api-playground"
+      context: ci/docker
+      kaniko:
+        useNewRun: true
+        singleSnapshot: true
+        snapshot-mode: "time"
+        buildArgs:
+          NUXEO_VERSION: "{{.NUXEO_VERSION}}"
+          BUILD_TAG: "{{.BUILD_TAG}}"
+          SCM_REF: "{{.GIT_COMMIT}}"
+          VERSION: "{{.VERSION}}"
+  cluster:
+    namespace: platform
+    dockerConfig:
+      secretName: jenkins-docker-cfg
+    resources:
+      requests:
+        cpu: "1"
+        memory: "3Gi"
```

Comment:
Should we try having request=limit?

Reply:
Yes we could give a first try on this.
I'll set:
```yaml
    resources:
      requests:
        cpu: "1"
        memory: "4Gi"
      limits:
        memory: "4Gi"
```
Which meet the best practices, Memory request == limit, and no CPU limit. Furthermore these settings follows what we have on the underlying Kubernetes nodes, ie: Memory = 4 x CPU.